web_widget_bokeh_chart/README.rst

```diff
@@ -27,7 +27,7 @@ Installation
 
 You need to install the python bokeh library::
```

Comment:
The example function refers to `p.html.data`. This does not seem to work, perhaps readme should be updated here as well so that we know how to fetch the data from the graph

Reply:
v12 is up-to-date: https://github.com/OCA/web/tree/12.0/web_widget_bokeh_chart. With that syntax it should work